dist/t/spec/features/0040_package_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "spec_helper"
+
+RSpec.describe "Package" do
+
+  it "should be able to create new" do
+    login('Admin','opensuse')
```

Comment:
I would put this line in a `before` block. And also all the other `login` lines in this file.